src/main/resources/category/kotlin/common.xml

```diff
@@ -153,7 +153,7 @@ class Foo {
 //ImportHeader[.//T-Identifier[@Text='java'] and .//T-Identifier[@Text='text']][1]/../..
 
 (: if used in setDateFormat function, allow if jackson ObjectMapper or XmlMapper is used :)
-//SimpleIdentifier//T-Identifier[(@Text="SimpleDateFormat") and ancestor::ClassMemberDeclaration
+//ClassMemberDeclaration//SimpleIdentifier//T-Identifier[(@Text="SimpleDateFormat")
```

Comment:
Je kunt dan toch ook zonder de ancestor af? 
//ClassMemberDeclaration[.//SimpleIdentifier//T-Identifier[(@Text="SimpleDateFormat")
and not (.//T-Identifier[@Text="setDateFormat"] and ...) ]

Reply:
Not sure if I understand. If I start with 
```//ClassMemberDeclaration[.//SimpleIdentifier//T-Identifier[(@Text="SimpleDateFormat")]]```
it matches `private` instead of `SimpleDateFormat`. How to go from there?